common/simulation.cpp

```diff
@@ -40,9 +40,9 @@ namespace tec {
 		this->event_list.mouse_click_events.clear();
 
 		GameState client_state = interpolated_state;
-		std::future<std::set<eid>> phys_future = std::async(std::launch::async, [=, &interpolated_state]() -> std::set < eid > {
+		std::future<std::set<eid>> phys_future = std::async(std::launch::async, [=, &interpolated_state] () -> std::set < eid > {
 			return phys_sys.Update(delta_time, interpolated_state);
-		});
+															});
```

Comment:
automatic indentation?

Reply:
It would seem so, or a kid/cat did it without me looking.